### ISSUE TITLE
Unet modelstore details

### ISSUE THREAD

**Issue by RSly** [NVIDIA/DIGITS#1858]
hi 
Do you know where I could get more info on the unet implementation ?
for example it says 'We expect 572 x 572 x 1'
it would be great to know why, and also what was the best accuracy, which hyper parameters, etc.

thanks

**Comment by ethantang95** [comment]
Hi, the input for UNet are images with resolution 572 by 572. This is with respect to the original paper for UNet.

For the best accuracy... it really varies. I believe the pretrained model used Adam with exponential decreasing training rate at 0.98 decay over 300 epochs or something, but that is because the dataset is small.

To find the best hyper parameters, it really is a trial and error problem.

@IsaacYangSLA Any comments on this?

**Comment by RSly** [comment]
thanks @ethantang95
So it seems that the number of classes are hard coded to be 2 in this implementation.
any simple way to make it multi-class unet? I see in the classification examples of digits+TF, the number of classes are automatically inferred from the data, which is great!

**Comment by ethantang95** [comment]
you can always change the unet network definition itself when you download the pretrained model. This implementation was done with respect to the paper but you are definitely free to experiment with it.

**Comment by MiralPatel3011** [comment]
when i download a pretrained unet model from model store and create model with my datasets gives error " : Message type "caffe.NetParameter" has no field named "from".".please gives me suggestion. Thank you.
